### PR TITLE
Beato d10 4 5

### PR DESCRIPTION
**Pull request by Avuxo**
:fujino:

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 -- Page 10012, Offset 25003.
 -- 午後２時を過ぎ、遅めの昼食を摂る事になった。
-It's past 2:00PM, and we decide to have a late lunch.
+It's past 2:00PM now, so we decide to have a late lunch.
 }
 [sha:47522d74370e9590e297a847bb7f1bc9ccc94c45]{
 -- Page 10012, Offset 25004.
 -- ……アルクェイドといる以上、いつもより高めの店を選びたかったのだが、悲しいかな、こちらの財布事情はそこまで裕福ではなく、
-...Since Arcueid's here, I wanted to choose a restaurant more expensive than usual, but alas, the condition of my wallet did not allow such luxury.
+...Since Arcueid's here, I wanted to choose a restaurant more expensive than usual, but alas, the conditions in my wallet allow not for such luxury.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:9]: I think i prefer the original here but "did not allow for such luxury"

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 2]
```diff
@@ -471,12 +480,13 @@ It's true that if I consume nutrients like this, then I can operate on my own st
 [sha:5d5fb2cee3390074e49b22b0f4eecf71773c3f4c]{
 -- Page 334, Offset 25097.
 -- 「わたしの熱量の補充方法は志貴たちとは違う。たしかに食欲はあるけど、それは人間で言うところの性欲と同じかな。
-"My method of replenishing calories is different than people like you. Sure, I can have an appetite, but it's most likely similar to the so-called sex drive for human beings.
+"My method of replenishing calories is different than yours. Sure, I have an appetite, but it's more like to the so-called sex drive in human beings.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:483]: drop to

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 3]
```diff
@@ -571,17 +582,18 @@ Even for me, I see it as something delicious even when I know it's poisonous."
...
 -- Page 342, Offset 25117.
 -- 摂らなくてもいいものを摂りたい、という欲求において、わたしと<人間|あなた>は変わらないわ」
-We're the same in that we both have desires for taking things that we don't need to take."
+We're the same in that we both have desires for having things that we don't need to have."
 }
 [sha:9c5cc3cc35632117c663d4409fc98f3915fca653]{
 -- Page 343, Offset 25118.
 -- 「ほら。知性体は繁殖であれ食事であれ、それに何らかの二次的利益を付加しないとさぼってしまう生き物でしょう？
-"I mean, whether it's about food or breeding, we creatures would avoid it if it doesn't provide some secondary benefit, right?
+"I mean, whether it's we're talking food or we're talking mating, we creatures would avoid it if it doesn't provide some secondary benefit, right?
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:590]: drop it's

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 4]
```diff
@@ -791,22 +804,23 @@ My body moves unintentionally from the shock.
...
 -- Page 355, Offset 25162.
 -- 俺はそのように考える吸血鬼を殺す為にここにいる。
-I came here to kill a vampire who thinks like that.
+I'm here to kill a vampire who thinks like that.
 }
 [sha:5d87cd2e739e55c00a7313961ff90c4323643c8b]{
 -- Page 355, Offset 25163.
 -- ……アルクェイドを、やつらと同類だと認める為にいるんじゃない。
-...I'm not here to accept that Arcueid is the same as that vampire.
+// double check.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:817]: looks fine

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 5]
```diff
@@ -791,22 +804,23 @@ My body moves unintentionally from the shock.
...
+...I'm not here to deem Arcueid an accomplice to that thought process.
 }
 [sha:923481f35ac4543bb2e5404bd88ef9ceae26a1fe]{
 -- Page 356, Offset 25164.
 -- 「……そりゃあ生きるために他の生き物を食うのは自然の摂理だろう。けど、人間とおまえは似たようなモンじゃないか。同種を食べるのが禁忌なのも自然界のルールだろう。
-"...Well, eating other animals in order to survive is the natural order. But aren't you similar to other humans? It would be taboo to eat one of your own kind under the rule of nature, right?
+"...Well, eating other animals in order to survive is the natural order. But aren't you similar humans? It would be taboo to eat one of your own kind under the rule of nature, right?
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:823]: to

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 6]
```diff
@@ -826,12 +840,13 @@ I don't like talking about what might happen.
...
 -- 「そう？　わたしはイフって好きよ。
-"Is that so? But I like talking about what-ifs.
+"Is that so? But I like talking them.
 }
 [sha:e2e7d23a162cb0375e3adf2945f3a23d1c9f98b4]{
 -- Page 357, Offset 25169.
 -- どんな結果になるか分からないけど、とりあえずその時は、どこかに救いがあるような気がするでしょう？」
-I don't know what would come as a result, but for now, you would feel like there's hope out there somewhere at that time, right?"
+// what?
+I don't know what might happen as a result, but in the moment, it feels like salvation is coming from somewhere, right?"
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:849]: idk man

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 7]
```diff
@@ -1196,12 +1213,12 @@ She suppresses her emotions from violently breaking out from within in some way.
 [sha:a29f2e4edf03ae504d9a740d0dd53948e27f98e4]{
 -- Page 385, Offset 25242.
 -- 名前が知れただけで充分だ。あとは直接見つけて、この目で確かめればいいさ」
-Knowing his name is more than enough. Now all we have to do is find him in person and confirm it with these eyes."
+Knowing his name is more than enough. Now all we have to do is find him in person and confirm it with my eyes."
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:1216]: my own eyes? same thing but i just like it imo

File: script/Arcueid/Day 10/10_00_ARC10_5.txt [hunk 8]
```diff
@@ -541,37 +546,38 @@ But is something being pointless really that bad?"
...
 -- Page 436, Offset 25374.
 -- 「……その思い出自体が無駄な事なのに、それを楽しく思い出せるっていうの、志貴は？」
-"...Even though those memories itself are pointless, are you saying that you enjoy thinking back on them?"
+"...Even though those memories themselves are pointless, are you saying that you'd enjoy thinking back on them?"
 }
 [sha:b8634934d1e767b9c1176d21e400f1a56623ebe8]{
 -- Page 437, Offset 25375.
 -- 「ああ、そのあたりは大丈夫。自分にとって都合の悪いものは思い出せないように出来てるんだよ、人間って。
-"Yeah, it's fine like that. Even I am able to forget things that are inconvenient, just like every other human.
+"Yeah, that's just fine. Even I'm able to forget things the inconveniences of the past, just like every other human.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_5.txt:569]: drop things

**Review by nolo16 — APPROVED** [review]
otsu

File: script/Arcueid/Day 10/10_00_ARC10_4.txt (hunk 7 above)
Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_4.txt:1216]: that reads pretty stilted imo

**Pull request merged**